test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/LandingPageControllerSpec.scala

```diff
@@ -46,8 +46,16 @@ class LandingPageControllerSpec extends ControllerSpec {
 
       checkPageIsDisplayed(
         result,
-        messageFromMessageKey("agentsLandingPage.title"),
-        doc => doc.select(".button").attr("href") shouldBe s"${viewConfig.agentsSignInUrl}"
+        messageFromMessageKey("agentsLandingPage.title"), doc => {
+          doc.select(".button").attr("href") shouldBe s"${viewConfig.agentsSignInUrl}"
```

Comment:
`s"${x}"` is equivalent to just `x` when `x` is a String. You could save a few characters here and below

Reply:
done, thank you - didn't spot that, just copied what was already there